ee/modules/650-runtime-audit-engine/templates/flowcontrol.yaml

```diff
@@ -0,0 +1,39 @@
+---
+apiVersion: flowcontrol.apiserver.k8s.io/v1beta1
```

Comment:
Denys has made some dynamic apiVersion template here. Because v1beta1 will be remove in the k8s 1.26

Reply:
AFAIK this PR will be refactored with no helper included.

Reply:
PR is closed.